events/janus_mqttevh.c

```diff
@@ -522,7 +550,7 @@ static int janus_mqttevh_init(const char *config_path) {
 	}
 	/* Read configuration */
 	char filename[255];
-	g_snprintf(filename, 255, "%s/%s.cfg", config_path, JANUS_MQTTEVH_PACKAGE);
+	g_snprintf(filename, 255, "%s/%s.jcfg", config_path, JANUS_MQTTEVH_PACKAGE);
```

Comment:
You're missing the fallback to `.cfg` that the other plugins have. This is actually my fault, but since you're fixing this part as well, the fallback should be added here.

Reply:
Okay, no problem.

Reply:
Added fallback and made config parsing more robust.